src/Tag/Media/Track.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Html\Tag\Media;
+
+use Yiisoft\Html\Tag\Base\VoidTag;
+
+/**
+ * @link https://html.spec.whatwg.org/multipage/media.html#the-track-element
+ */
+final class Track extends VoidTag
+{
+    public function isDefault(): bool
+    {
+        if (isset($this->attributes['default'])) {
+            return $this->attributes['default'] !== false;
+        }
+
+        return false;
+    }
+
+    public function default(bool $default = true): self
+    {
+        return $this->attribute('default', $default);
+    }
+
+    public function kind(string $kind): self
```

Comment:
1) Suggest add kind variants as constants: `Track::SUBTITLES`, `Track::CAPTIONS` and etc.

2) `kind` is not required attribute, maybe allow pass `null`?